CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Version 4.2.1 (Under development)
 
+### App Center Distribute Play
+
+- **[Fix]** #1515, Xamarin.Forms Android Package built with `Microsoft.AppCenter.DistributePlay` contains `Microsoft.AppCenter.Distribute.Android.Bindings.dll`.
```

Comment:
Thanks for your contribution PR!

Could you please describe what exactly fixes this PR and remove the number of the issue (see how the other fixes are described in changelog)? 

If I right understand you are planning to add the `DistributePlay` module to both projects (common and Android) to avoid this issue. I checked your changes locally and for Android, it works well, but it broke some behavior for iOS because `DistributePlay` doesn't contain the [library](https://github.com/microsoft/appcenter-sdk-dotnet/blob/develop/nuget/AppCenterDistribute.nuspec#L36) for iOS. 
But I think we take care about it ourselves.

Reply:
I will continue this in 1-2 days.